src/pages/list/CardListPage.jsx

```diff
@@ -159,67 +79,61 @@ function CardListPage() {
-                ),
-              )}
-            {/* </Slider> */}
-          </div>
-          <div
-            className={`arrow right ${hotOffset + limit < hotRecipients.length ? '' : 'disabled'}`}
-            onClick={() => hotListShift(2)}
-          >
-            <ArrowButton direction={'right'} />
+            <div
+              className={`arrow left ${hotOffset > 0 ? '' : 'disabled'}`}
+              onClick={() => hotListShift(-2)}
+            >
+              <ArrowButton direction={'left'} />
+            </div>
+            {sliceWithFallback(hotRecipients, hotOffset, limit).map((data) => (
+              <CardList key={`${data.id}`} recipient={data} />
+            ))}
+            <div
+              className={`arrow right ${hotOffset + limit < hotRecipients.length ? '' : 'disabled'}`}
+              onClick={() => hotListShift(2)}
+            >
+              <ArrowButton direction={'right'} />
+            </div>
           </div>
         </div>
       </div>
       <div className='card-list-box'>
+        <div className='title-layer'>
         <p className='card-list-title'>최근에 만든 롤링 페이퍼 ⭐</p>
+        </div>
         <div className='card-list-data'>
-          <div
-            className={`arrow left ${offset > 0 ? '' : 'disabled'}`}
-            onClick={() => listShift(-2)}
-          >
-            <ArrowButton direction={'left'} />
-          </div>
           <div className='card-list-wrapper date-card' ref={dateListRef}>
+            <div
+              className={`arrow left ${offset > 0 ? '' : 'disabled'}`}
+              onClick={() => listShift(-2)}
+            >
+              <ArrowButton direction={'left'} />
+            </div>
             {sliceWithFallback(recipients, offset, limit).map((data) => (
               <CardList key={`${data.id}`} recipient={data} />
             ))}
-          </div>
-          <div
-            className={`arrow right ${offset + limit < recipients.length ? '' : 'disabled'}`}
-            onClick={() => listShift(2)}
-          >
-            <ArrowButton direction={'right'} />
+            <div
+              className={`arrow right ${offset + limit < recipients.length ? '' : 'disabled'}`}
+              onClick={() => listShift(2)}
+            >
+              <ArrowButton direction={'right'} />
+            </div>
           </div>
         </div>
       </div>
-      <Button
-        children='나도 만들어보기'
-        size={56}
-        className={'post-link-button'}
-        handleClick={() => handleMovePage('/post')}
-      />
-      {/* <div className='testlayer'>
-        <br />
-        <TextDropdownField options={test}></TextDropdownField>
-        <br />
-        <br />
-        <TextInputField></TextInputField>
-        <br />
-      </div> */}
+      <div className='link-button-layer'>
+        <Button
+          children='나도 만들어보기'
+          size={56}
+          className={'post-link-button'}
+          handleClick={() => handleMovePage('/post')}
+        />
```

Comment:
```suggestion
<Button
          size={56}
          className={'post-link-button'}
          handleClick={() => handleMovePage('/post')}
        >
나도 만들어보기
</Button>
```
children은 html태그 쓰던거처럼 안에 써주면 알아서 적용돼용 ~ 저렇게 prop으로 직접 전달해주는게 아니라